netbox/dcim/views.py

```diff
@@ -652,6 +652,7 @@ def get(self, request):
 
         racks = filtersets.RackFilterSet(request.GET, self.queryset).qs
         total_count = racks.count()
+        self.model = self.queryset.model
```

Comment:
We can avoid setting the model on the instance and just pass it directly as context for the template.
```suggestion
```